specification/specs/x_pack/async_search/get/AsyncSearchGetRequest.ts

```diff
@@ -25,7 +25,6 @@
 interface AsyncSearchGetRequest extends RequestBase {
   path_parts?: {
     id: Id
-    typed_keys?: boolean
```

Comment:
This should be moved to `query_parameters`, with `wait_for_completion_timeout` and `keep_alive`.

Reply:
This has been there for a while and there are many similar validation errors. Let's address all of them in a separate PR.